src/pretix/presale/views/order.py

```diff
@@ -602,13 +602,18 @@ def get_confirm_url(self, payment):
 
 
 def can_generate_invoice(event, order, ignore_payments=False):
```

Comment:
I changed this method so when changing to bank transfer the customer can request an invoice. It is not automatically generated in this case.